Formula/grin-wallet.rb

```diff
@@ -0,0 +1,17 @@
+class GrinWallet < Formula
+  desc "Grin Wallet, official wallet for the cryptocurrency Grin"
```

Comment:
How about:
```suggestion
  desc "Official wallet for the cryptocurrency Grin"
```

Otherwise it's repetitive